### PR TITLE
Reviews saving to db and returning to front end

### PR DESCRIPTION
**Pull request by cmonteilh**

### CODE REVIEW

**Review by stevenharderjr — APPROVED** [review]
Taught me something knew about React. That's always a code review bonus.

**Comment by PBandJaimie** [comment]
I'm not sure what the issue is with the vulnerability in sonar, but this code looks amazing. I'm so excited to merge it and see it go! Thank you @cmonteilh!

File: client/features/reviews/ReviewCard.jsx [hunk 1]
```diff
@@ -3,20 +3,17 @@ import StarRatingInteractive from '../sharedComponents/StarRatingInteractive';
 import Reviews from '../reviews/Reviews.jsx';
 import './../media/MediaDetail.css';
 
-const ReviewCard = props => {
-  const { userId, username, starRating, userReview, key } = this.props;
+const ReviewCard = review => {
+  const { username, rating, content } = review;
```
[the remark this answers is not in the document]
Comment (cmonteilh) [reply client/features/reviews/ReviewCard.jsx:7]: Thanks!

File: client/features/reviews/Reviews.jsx [hunk 2]
```diff
@@ -2,93 +2,101 @@ import React from 'react';
...
     this.setState({ userReview: e.target.value });
   }
 
-  handleSubmit() {
-    const { mediaId, userId } = this.props;
-    axios.post('/submit-review', {
+  submitReview(e) {
+    e.preventDefault();
+    const { mediaId, username, userReview } = this.state;
+    axios.post('/reviews/submit-review', {
```
[the remark this answers is not in the document]
Comment (cmonteilh) [reply client/features/reviews/Reviews.jsx:56]: Ah yes, you're correct. I wrote it this way to be consistent with how the other components wrote their routes.

**Comment by cmonteilh** [comment]
Thank you @PBandJaimie and @stevenharderjr !

**Pull request merged**